### ISSUE TITLE
Clear icon is shown when disabled

### ISSUE THREAD

**Issue by web-padawan** [vaadin/vaadin-combo-box#407]
### Description
If `vaadin-combo-box` has a value selected but is disabled, clear icon is shown (and expand icon is not). This looks buggy, why to show clear icon if we can't clear?

### Expected outcome
Icon should be hidden, e. g. 
```css
      :host([disabled]) paper-icon-button.clear-button,
      :host([disabled]) ::content paper-icon-button.clear-button  {
        display: none;
      }
```

### Actual outcome
Icon stays visible while combo box is disabled.

### Live Demo
Example: https://jsbin.com/veligoviza/1/edit?html,output

**Comment by Saulis** [comment]
Thanks for the report, that's definitely a bug.

**Comment by jay8t6** [comment]
+1 I was just about to report this bug

**Comment by Saulis** [comment]
Released the fix in `v1.3.1`, https://github.com/vaadin/vaadin-combo-box/releases/tag/v1.3.1

**Comment by jay8t6** [comment]
Great, thanks!
